great_expectations/rule_based_profiler/helpers/util.py

```diff
@@ -557,9 +564,14 @@ def compute_bootstrap_quantiles_point_estimate(
             bootstrap_upper_quantile_point_estimate - bootstrap_upper_quantile_bias
         )
 
-    return (
-        lower_quantile_bias_corrected_point_estimate,
-        upper_quantile_bias_corrected_point_estimate,
+    return NumericRangeEstimationResult(
+        estimation_histogram=np.histogram(
+            a=bootstraps.flatten(), bins=NUM_HISTOGRAM_BINS
```

Comment:
This is tricky due to the "custom" nature of our implementation. We should discuss this. The bootstraps are not what is expected to be normal, the computation of the statistic on the bootstraps is.

Reply:
@alexsherstinsky don't our bootstraps consist of the samples of the statistic, which is what we expect to be `Normal`?